stdlib/sqlite3/dbapi2.pyi

```diff
@@ -227,9 +227,9 @@ else:
 
 if sys.version_info < (3, 8):
     class Cache:
-        def __init__(self, *args, **kwargs) -> None: ...
-        def display(self, *args, **kwargs) -> None: ...
-        def get(self, *args, **kwargs) -> None: ...
+        def __init__(self, *args: Incomplete, **kwargs: object) -> None: ...
```

Comment:
Here's `kwargs` is `object` because `kwargs` is not used in `Cache` constructor: https://github.com/python/cpython/blob/b0b590be9597fd5919228d251812dd54145f70a7/Modules/_sqlite/cache.c#L57-L87